docs/en/writing-running-appium/caps.md

```diff
@@ -167,6 +167,7 @@ These Capabilities are available only on the [Espresso Driver](/docs/en/drivers/
 |`espressoServerLaunchTimeout`|Timeout in milliseconds used to wait for the Espresso server to launch. Defaults to `30000` |e.g., `50000`|
 |`espressoBuildConfig`|Path to the Espresso server build configuration JSON (see below)|e.g., `/projects/myapp-tests/buildconfig.json`|
 |`showGradleLog`|Whether to pipe Gradle build log for the Espresso server to the Appium log. Defaults to `false`|e.g., `true`|
+|`intentOptions`|Intent options which will be used to start the application under test. It can set intent options such as `action`, `categories` and `flags` as JSON format. Please read [#538](https://github.com/appium/appium-espresso-driver/issues/538) as an example usage. | e.g. `{"es": ["test data as string"], "action": "android.intent.action.MAIN", "categories": "android.intent.category.LAUNCHER", "component": "com.appium/.launcher.MainActivity", "flags": "FLAG_ACTIVITY_NEW_TASK"}` |
```

Comment:
I'd rather add this option later after @rajdeepv creates more detailed documentation for it.

Reply:
Yeah, great to add as another documentation here

Reply:
I have added a paragraph about this in https://github.com/appium/appium/pull/13901
Not sure how much detailed should this be, please let me know